app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] The application should ask for confirmation of email, the `:confirmable` should be enabled in the user model 🙃 🙃 
- [OPTIONAL] I think it can be better if you style your signup page as you styled your login page 👍 
- [ ] After login into your app and when trying to create a post nothing happens, I can't create a post. And if I can't create a post I can't know if it's possible to add a comment and like as well. Please kindly make sure your app work as expected 🙃 🙃

Reply:
Hello @cynthiainga, Thank you for the review. I have implemented the confirmable and everything is working. 😄

Reply:
Issue with the post not working is resolved. 😄